packages/chart-advisor/src/advisor/index.ts

```diff
@@ -123,19 +123,28 @@ export class Advisor {
       Object.keys(data).some((key) => keyForGraph.includes(key));
     const shouldRecommendGraph =
       (options as GraphAdviseParams['options'])?.extra || purposeForGraphs.includes(options?.purpose) || hasKeyForGraph;
-    let advices;
+
+    let adviceResult: Advice[] | AdvicesWithLog;
+
     const graphAdvices = this.advicesForGraph(params as GraphAdviseParams);
     // const chartAdvices = this.advicesForChart(params as ChartAdviseParams);
     // If shouldRecommendGraph is true, higher priority for relational graph
     if (shouldRecommendGraph) {
-      advices = graphAdvices;
+      adviceResult = graphAdvices;
       // advices = graphAdvices.concat(chartAdvices)
     } else {
       const chartAdvices = this.advicesForChart(params as ChartAdviseParams);
       // Otherwise, higher priority for statistical charts
-      advices = chartAdvices.concat(graphAdvices);
+      if ((options as ChartAdviseParams['options'])?.exportLog) {
```

Comment:
is there an `exportLog` option in GraphAdviseParams['options']?

Reply:
not yet...we would refactor graph-advise pipeline later